test/blackbox-tests/test-cases/enabled_if/enabled-if-lib-other-vars.t

```diff
@@ -0,0 +1,16 @@
+Here we demonstrate that we expand any variable in enabled_if on libraries
```

Comment:
Are there other tests that were showing the previous behavior? If that's the case, might be worth to move this case over there or put them together in some way?

Reply:
Yes, see the other tests updated in this PR. The previous behavior is still preserved (although the error message is tweaked)